collector/serverstats.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 Ben Kochie
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"fmt"
+
+	"github.com/facebook/time/ntp/chrony"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	serverstatsSubsystem = "serverstats"
+)
+
+var (
+	serverstatsNTPHits = typedDesc{
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, serverstatsSubsystem, "ntp_hits"),
```

Comment:
All metrics that are counters must be named ending with `_total`.

Also, since the name of the metric struct / internal name and documentation differ slightly, I'd prefer the documentation name, since it reflects what this data is about.

What do you think of this:

```suggestion
			prometheus.BuildFQName(namespace, serverstatsSubsystem, "ntp_received_packets_total"),
```